sites/platform/src/domains/cdn/cloudflare.md

```diff
@@ -33,20 +33,28 @@ through your DNS provider.
 But `CNAME` records can't point to apex domains.
 As a workaround, Cloudflare offers [`CNAME` flattening](https://developers.cloudflare.com/dns/additional-options/cname-flattening/).
 
-## 4. Optional: Protect your site from on-path attacks
-
-An on-path attack occurs when a hacker intercepts or modifies the communication between a client and a server.
-This can lead to sensitive data leaks.
-To prevent such attacks, make sure all communication with your site is encrypted through HTTPS
-and can't be downgraded to HTTP.
-
-To do so, [enable full strict SSL/TLS encryption](https://developers.cloudflare.com/ssl/origin-configuration/ssl-modes/full-strict/).
-Any communication between a client and Cloudflare
-or between Cloudflare and your {{% vendor/name %}} server is then encrypted through HTTPS.
-In addition, Cloudflare checks that your {{% vendor/name %}} server's [TLS certificate](/glossary.md#transport-layer-security-tls) 
-was issued by a trusted certificate authority.
-This confirms the client is truly communicating with your {{% vendor/name %}} server.
-
-For enhanced security, make sure your HTTPS connections can't be downgraded to HTTP.
-To do so, in your Cloudflare account,
-[enable HTTP strict transport security (HSTS)](https://developers.cloudflare.com/ssl/edge-certificates/additional-options/http-strict-transport-security/).
+## 4. Mitigate security risks
+
+Like all networks exposed to the internet, your CDN may become the target of security attacks.
```

Comment:
Need to change `your CDN may become the target` with `your origin server may become the target`.

Both security mechanism help ensuring that either: requests cannot be sent to the origin, or requests coming from the CDN to the origin are verify difficult to temper with.